prog/probe.go

```diff
@@ -266,9 +266,7 @@ func probeMain(flags probeFlags, targets []appclient.Target) {
 	}
 
 	if flags.dockerEnabled {
-		// Don't add the bridge in Kubernetes since container IPs are global and
-		// shouldn't be scoped
-		if flags.dockerBridge != "" && !flags.kubernetesEnabled {
+		if flags.dockerBridge != "" {
```

Comment:
This is an improvement for when we have "unmanaged" containers running on a k8s host. However, it will also result in

1. spurious error messages when docker isn't running on a k8s host (as will happen by default on some future k8s releases)

2. incorrect treatment of the docker bridge as local if it is in fact the bridge used by k8s

We could avoid the first one by only erroring when the bridge name has been specified on the command line (rather than left to the default 'docker0' value).

Is the 2nd problem real?

Reply:
2. I don't think it is practical to use the `docker0` bridge as the Kubernetes bridge nowadays except on a single-node cluster.
I think Flannel used to connect one machine's Docker bridge to another, but nowadays that is driven via CNI with its own bridge.  And Docker's own overlay networking will create a different bridge for each network.